source/mapping-rules/introduction.txt

```diff
@@ -39,17 +37,16 @@ documentation:
 - `Summary of Schema Design Anti-Patterns and How to Spot Them
   <https://www.mongodb.com/developer/products/mongodb/schema-design-anti-pattern-summary/>`__.
 
-Mapping Direction Preference
-----------------------------
+Mapping Rules
+-------------
 
-The source relational model is static and unchanging while the target MongoDB model
-is dynamic and customizable.
-
-You can create a mapping rule based on a source :ref:`relational model <create-rule-relational>`
-or based on a destination :ref:`MongoDB model <create-rule-mongodb>`.
+A mapping rule specifies where data from each relational table should be migrated to and how the 
```

Comment:
To avoid "should" and write this slightly more directly, maybe we can write this as:

"Mapping rules specify how relational table columns are mapped to MongoDB documents."